docs/content/contributing/community/office-hours.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Developer Meetings"
```

Comment:
Should the title change to 'Office Hours' as well?

Reply:
Yes! Good catch, changing now